src/providers/DocumentSymbolProvider.ts

```diff
@@ -91,13 +89,13 @@ export class DocumentSymbolProvider implements vscode.DocumentSymbolProvider {
 							var as_loc   = type_var.toLowerCase().indexOf('as')
 							pieces = type_vars[x].trim().toLowerCase().split('as');
 							var child_symbol = type_var.substring(0, as_loc);
-							if (pieces.length >= 1 && pieces[1]) {
+							if (pieces.length >= 1 && pieces[1] && !child_symbol.startsWith("'")) {
 								var child_symbolText = type_var.substring(as_loc);
 								var child_symbolKind = vscode.SymbolKind.Property;
 								symbolChildren.push(
 									new vscode.DocumentSymbol(
-										child_symbol.trim() || ' ',
-										child_symbolText.trim() || ' ',
+										child_symbol.trim() || ' ', // Kludge "cannot be falsy"
```

Comment:
Same with these. 
This kludge is to just move on with life, and provide best effort. Ideally we would not have to do this, but, w/e for now.